Pre-released 4.1.2-SNAPSHOT.jar update fails on insertMany change type in a JSON changeLog

Pre-released 4.1.2-SNAPSHOT.jar update fails on insertMany change type in a JSON changeLog

**STR**
Build a 4.1.2-SNAPSHOT.jar locally with the latest changes that includes the formatted JSON changeLog/change-types.
Include liquibase-mongodb-4.1.2-SNAPSHOT.jar and mongo-java-driver-3.12.7.jar in {liquibase_home}/lib directory.

Run `liquibase update` with a an insertMany cahnge type in a JSON changeLog.

For example: https://github.com/liquibase/liquibase-mongodb/blob/main/src/test/resources/liquibase/ext/json/generic-1-insert-people.json

Expected result:
Liquibase should create the entries for the insertMany change type in the JSON changest/changelog

Actual result:
Liquibase fails with the following error:
`$ liquibase --changeLogFile=generic-1-insert-people.json --logLevel=debug update 
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.license.LicenseService instance liquibase.pro.license.keymgr.DaticalTrueLicenseService
[2020-11-01 15:31:11] INFO [liquibase.integration] No Liquibase Pro license key supplied. Please set liquibaseProLicenseKey on command line or in liquibase.properties to use Liquibase Pro features.
Liquibase Community 4.1.1 by Datical
Starting Liquibase at 15:31:11 (version 4.1.1 #10 built at 2020-10-12 19:24+0000)
[2020-11-01 15:31:11] FINE [liquibase.integration] Liquibase Hub URL:      https://hub.liquibase.com
[2020-11-01 15:31:11] FINE [liquibase.integration] Liquibase Hub Mode:     all
[2020-11-01 15:31:11] FINE [liquibase.resource] Adding path /Users/support.liquibase.net/CLI-Projects/MongoDB/. to resourceAccessor liquibase.resource.FileSystemResourceAccessor
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.DB2Database
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.Db2zDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.DerbyDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.Firebird3Database
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.FirebirdDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.H2Database
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.HsqlDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.InformixDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.Ingres9Database
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MSSQLDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MariaDBDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MockDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MySQLDatabase
[2020-11-01 15:31:11] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.OracleDatabase
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.PostgresDatabase
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.SQLiteDatabase
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.SybaseASADatabase
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.SybaseDatabase
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.UnsupportedDatabase
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.ext.mongodb.database.MongoLiquibaseDatabase
[2020-11-01 15:31:12] FINE [liquibase.database] Properties:
[2020-11-01 15:31:12] FINE [liquibase.database] Connecting to the URL:'mongodb://localhost:27017/MYDB' using driver:'liquibase.ext.mongodb.database.MongoClientDriver'
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.DatabaseConnection instance liquibase.database.jvm.JdbcConnection
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.DatabaseConnection instance com.datical.liquibase.ext.database.jvm.ProJdbcConnection
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.database.DatabaseConnection instance liquibase.ext.mongodb.database.MongoConnection
[2020-11-01 15:31:12] INFO [com.mongodb.diagnostics.logging.JULLogger log] Cluster created with settings {hosts=[localhost:27017], mode=SINGLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500}
[2020-11-01 15:31:12] FINE [liquibase.database] Connection has been created
[2020-11-01 15:31:12] FINE [liquibase.ext] Connected to null@localhost:27017
[2020-11-01 15:31:12] FINE [liquibase.ext] Not adjusting the auto commit mode; it is already false
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.Catalog
[2020-11-01 15:31:12] INFO [com.mongodb.diagnostics.logging.JULLogger log] Opened connection [connectionId{localValue:1, serverValue:35}] to localhost:27017
[2020-11-01 15:31:12] INFO [com.mongodb.diagnostics.logging.JULLogger log] Monitor thread successfully connected to server with description ServerDescription{address=localhost:27017, type=STANDALONE, state=CONNECTED, ok=true, version=ServerVersion{versionList=[4, 4, 1]}, minWireVersion=0, maxWireVersion=9, maxDocumentSize=16777216, logicalSessionTimeoutMinutes=30, roundTripTimeNanos=6888234}
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.Column
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.Data
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.ForeignKey
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.Index
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.PrimaryKey
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.Schema
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.Sequence
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.StoredProcedure
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.Table
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.UniqueConstraint
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance liquibase.structure.core.View
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance com.datical.liquibase.ext.appdba.synonym.Synonym
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance com.datical.liquibase.ext.storedlogic.checkconstraint.CheckConstraint
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance com.datical.liquibase.ext.storedlogic.databasepackage.DatabasePackage
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance com.datical.liquibase.ext.storedlogic.databasepackage.DatabasePackageBody
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance com.datical.liquibase.ext.storedlogic.function.Function
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.structure.DatabaseObject instance com.datical.liquibase.ext.storedlogic.trigger.Trigger
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.lockservice.LockService instance liquibase.lockservice.LockServiceImpl
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.lockservice.LockService instance liquibase.lockservice.MockLockService
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.lockservice.LockService instance liquibase.lockservice.OfflineLockService
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.lockservice.LockService instance liquibase.lockservice.StandardLockService
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.lockservice.LockService instance liquibase.ext.mongodb.lockservice.MongoLockService
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.executor.Executor instance liquibase.executor.jvm.JdbcExecutor
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.executor.Executor instance com.datical.liquibase.ext.OracleSqlPlusExecutor
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.executor.Executor instance liquibase.nosql.executor.NoSqlExecutor
[2020-11-01 15:31:12] INFO [com.mongodb.diagnostics.logging.JULLogger log] Opened connection [connectionId{localValue:2, serverValue:36}] to localhost:27017
[2020-11-01 15:31:12] INFO [liquibase.ext] Adjusting database Lock Collection with name: MYDB.DATABASECHANGELOGLOCK
[2020-11-01 15:31:12] INFO [liquibase.ext] Adjusted database Lock Collection with name: MYDB.DATABASECHANGELOGLOCK
[2020-11-01 15:31:12] INFO [liquibase.ext] Lock Database
[2020-11-01 15:31:12] INFO [liquibase.ext] Successfully Acquired Change Log Lock
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.parser.ChangeLogParser instance liquibase.parser.core.formattedsql.FormattedSqlChangeLogParser
[2020-11-01 15:31:12] FINE [liquibase.servicelocator] Loaded liquibase.parser.ChangeLogParser instance liquibase.parser.core.json.JsonChangeLogParser
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.parser.ChangeLogParser instance liquibase.parser.core.sql.SqlChangeLogParser
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.parser.ChangeLogParser instance liquibase.parser.core.xml.XMLChangeLogSAXParser
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.parser.ChangeLogParser instance liquibase.parser.core.yaml.YamlChangeLogParser
[2020-11-01 15:31:13] INFO [liquibase.servicelocator] Cannot load service: liquibase.parser.ChangeLogParser: liquibase.nosql.parser.json.JsonNoSqlChangeLogParser Unable to get public no-arg constructor
[2020-11-01 15:31:13] FINE [liquibase.resource] Closing duplicate stream for file:/Users/support.liquibase.net/CLI-Projects/MongoDB/generic-1-insert-people.json
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddAutoIncrementChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddColumnChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddDefaultValueChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddForeignKeyConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddLookupTableChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddNotNullConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddPrimaryKeyChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AddUniqueConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.AlterSequenceChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.CreateIndexChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.CreateProcedureChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.CreateSequenceChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.CreateTableChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.CreateViewChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DeleteDataChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropAllForeignKeyConstraintsChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropColumnChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropDefaultValueChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropForeignKeyConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropIndexChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropNotNullConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropPrimaryKeyChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropProcedureChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropSequenceChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropTableChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropUniqueConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.DropViewChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.EmptyChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.ExecuteShellCommandChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.InsertDataChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.LoadDataChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.LoadUpdateDataChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.MergeColumnChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.ModifyDataTypeChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.OutputChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.RawSQLChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.RenameColumnChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.RenameSequenceChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.RenameTableChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.RenameViewChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.SQLFileChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.SetColumnRemarksChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.SetTableRemarksChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.StopChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.TagDatabaseChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.core.UpdateDataChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.change.custom.CustomChangeWrapper
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.appdba.markunused.change.MarkUnsedChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.appdba.synonym.change.CreateSynonymChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.appdba.synonym.change.DropSynonymChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.AddCheckConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.DisableCheckConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.DropCheckConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.EnableCheckConstraintChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.databasepackage.change.CreatePackageBodyChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.databasepackage.change.CreatePackageChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.databasepackage.change.DropPackageBodyChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.databasepackage.change.DropPackageChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.function.change.CreateFunctionChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.function.change.DropFunctionChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.trigger.change.CreateTriggerChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.trigger.change.DisableTriggerChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.trigger.change.DropTriggerChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.trigger.change.EnableTriggerChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance com.datical.liquibase.ext.storedlogic.trigger.change.RenameTriggerChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.ext.mongodb.change.AdminCommandChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.ext.mongodb.change.CreateCollectionChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.ext.mongodb.change.CreateIndexChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.ext.mongodb.change.InsertManyChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.ext.mongodb.change.InsertOneChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.change.Change instance liquibase.ext.mongodb.change.RunCommandChange
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.changelog.ChangeLogHistoryService instance liquibase.changelog.MockChangeLogHistoryService
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.changelog.ChangeLogHistoryService instance liquibase.changelog.StandardChangeLogHistoryService
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.changelog.ChangeLogHistoryService instance liquibase.ext.mongodb.changelog.MongoHistoryService
[2020-11-01 15:31:13] INFO [liquibase.ext] Adjusting database history Collection with name: MYDB.DATABASECHANGELOG
[2020-11-01 15:31:13] INFO [liquibase.ext] Adjusted database history Collection with name: MYDB.DATABASECHANGELOG
[2020-11-01 15:31:13] FINE [liquibase.util] Computed checksum for createCollection:[
    collectionName="person"
] as 08850fdc8ad5be416f97add1312fb2e2
[2020-11-01 15:31:13] FINE [liquibase.util] Computed checksum for 8:08850fdc8ad5be416f97add1312fb2e2: as 977093896ec672227927b3b45b0d16bb
[2020-11-01 15:31:13] FINE [liquibase.util] Computed checksum for insertOne:[
    collectionName="person"
] as 988df75ad4fadf026ee78872503f44c4
[2020-11-01 15:31:13] FINE [liquibase.util] Computed checksum for insertMany:[
    collectionName="person"
] as 0344462153932e5f1676b84a57a03d69
[2020-11-01 15:31:13] FINE [liquibase.util] Computed checksum for 8:988df75ad4fadf026ee78872503f44c4:8:0344462153932e5f1676b84a57a03d69: as faa3c91ec8c8504a85993e79a19f4df1
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGenerator
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorDB2
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorHsqlH2
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorInformix
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorMySQL
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorSQLite
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddColumnGenerator
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddColumnGeneratorDefaultClauseBeforeNotNull
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddColumnGeneratorSQLite
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGenerator
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorDerby
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorInformix
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorMSSQL
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorMySQL
[2020-11-01 15:31:13] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorOracle
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorPostgres
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorSQLite
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorSybase
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorSybaseASA
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddForeignKeyConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddPrimaryKeyGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddPrimaryKeyGeneratorInformix
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddUniqueConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddUniqueConstraintGeneratorInformix
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddUniqueConstraintGeneratorTDS
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AlterSequenceGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.BatchDmlExecutablePreparedStatementGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ClearDatabaseChangeLogTableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CommentGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CopyRowsGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateDatabaseChangeLogLockTableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateDatabaseChangeLogTableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateDatabaseChangeLogTableGeneratorSybase
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateIndexGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateIndexGeneratorFirebird
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateIndexGeneratorPostgres
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateProcedureGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateSequenceGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateTableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateTableGeneratorInformix
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateViewGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateViewGeneratorInformix
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DeleteGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropColumnGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropDefaultValueGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropForeignKeyConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropIndexGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropPrimaryKeyGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropProcedureGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropSequenceGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropTableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropUniqueConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropViewGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetNextChangeSetSequenceValueGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorDB2
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorDerby
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorFirebird
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorHsql
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorInformix
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorMSSQL
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorOracle
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorPostgres
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorSybase
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorSybaseASA
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InitializeDatabaseChangeLogLockTableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertDataChangeGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorDB2
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorH2
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorHsql
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorInformix
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorMSSQL
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorMySQL
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorOracle
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorPostgres
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorSQLite
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorSybaseASA
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertSetGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.LockDatabaseChangeLogGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.MarkChangeSetRanGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ModifyDataTypeGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RawSqlGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ReindexGeneratorSQLite
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RemoveChangeSetRanStatusGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameColumnGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameSequenceGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameTableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameViewGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ReorganizeTableGeneratorDB2
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RuntimeGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SelectFromDatabaseChangeLogGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SelectFromDatabaseChangeLogLockGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetColumnRemarksGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetNullableGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetTableRemarksGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.StoredProcedureGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.TableRowCountGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.TagDatabaseGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UnlockDatabaseChangeLogGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UpdateChangeSetChecksumGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UpdateDataChangeGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UpdateGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.appdba.markunused.change.MarkUnusedGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.appdba.synonym.change.CreateSynonymGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.appdba.synonym.change.DropSynonymGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.AddCheckConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.DisableCheckConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.DropCheckConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.EnableCheckConstraintGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.CreatePackageBodyGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.CreatePackageGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.DropPackageBodyGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.DropPackageGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.function.change.CreateFunctionGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.function.change.DropFunctionGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.CreateTriggerGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.DisableTriggerGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.DropTriggerGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.EnableTriggerGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.RenameTriggerGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.nosql.executor.NoSqlGenerator
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.hub.HubService instance liquibase.hub.core.MockHubService
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.hub.HubService instance liquibase.hub.core.OnlineHubService
[2020-11-01 15:31:14] FINE [liquibase.servicelocator] Loaded liquibase.hub.HubService instance liquibase.hub.core.OfflineHubService
[2020-11-01 15:31:14] FINE [liquibase.changelog] Running Changeset:generic-1-insert-people.json::2::Nick
[2020-11-01 15:31:14] INFO [liquibase.nosql] Changeset generic-1-insert-people.json::2::Nick
[2020-11-01 15:31:14] INFO [liquibase.nosql] Populate person table
[2020-11-01 15:31:14] FINE [liquibase.changelog] Reading ChangeSet: generic-1-insert-people.json::2::Nick
[2020-11-01 15:31:14] INFO [liquibase.nosql] WARNING The following SQL may change each run and therefore is possibly incorrect and/or invalid:
[2020-11-01 15:31:14] FINE [liquibase.ext] Executing Statement: db.person.insertOne({}, {});
[2020-11-01 15:31:14] INFO [liquibase.ext] Release Database Lock
[2020-11-01 15:31:14] INFO [liquibase.ext] Successfully released change log lock
[2020-11-01 15:31:14] INFO [com.mongodb.diagnostics.logging.JULLogger log] Closed connection [connectionId{localValue:2, serverValue:36}] to localhost:27017 because the pool has been closed.
Unexpected error running Liquibase: Migration failed for change set generic-1-insert-people.json::2::Nick:
     Reason: java.lang.NullPointerException
For more information, please use the --logLevel flag
[2020-11-01 15:31:14] SEVERE [liquibase.integration] Unexpected error running Liquibase: Migration failed for change set generic-1-insert-people.json::2::Nick:
     Reason: java.lang.NullPointerException
liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set generic-1-insert-people.json::2::Nick:
     Reason: java.lang.NullPointerException
        at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:124)
        at liquibase.Liquibase.lambda$null$0(Liquibase.java:275)
        at liquibase.Scope.lambda$child$0(Scope.java:160)
        at liquibase.Scope.child(Scope.java:169)
        at liquibase.Scope.child(Scope.java:159)
        at liquibase.Scope.child(Scope.java:138)
        at liquibase.Scope.child(Scope.java:222)
        at liquibase.Liquibase.lambda$update$1(Liquibase.java:274)
        at liquibase.Scope.lambda$child$0(Scope.java:160)
        at liquibase.Scope.child(Scope.java:169)
        at liquibase.Scope.child(Scope.java:159)
        at liquibase.Scope.child(Scope.java:138)
        at liquibase.Liquibase.runInScope(Liquibase.java:2277)
        at liquibase.Liquibase.update(Liquibase.java:215)
        at liquibase.Liquibase.update(Liquibase.java:201)
        at liquibase.integration.commandline.Main.doMigration(Main.java:1760)
        at liquibase.integration.commandline.Main$1.lambda$run$0(Main.java:361)
        at liquibase.Scope.lambda$child$0(Scope.java:160)
        at liquibase.Scope.child(Scope.java:169)
        at liquibase.Scope.child(Scope.java:159)
        at liquibase.Scope.child(Scope.java:138)
        at liquibase.Scope.child(Scope.java:222)
        at liquibase.Scope.child(Scope.java:226)
        at liquibase.integration.commandline.Main$1.run(Main.java:360)
        at liquibase.integration.commandline.Main$1.run(Main.java:193)
        at liquibase.Scope.child(Scope.java:169)
        at liquibase.Scope.child(Scope.java:145)
        at liquibase.integration.commandline.Main.run(Main.java:193)
        at liquibase.integration.commandline.Main.main(Main.java:156)
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set generic-1-insert-people.json::2::Nick:
     Reason: java.lang.NullPointerException
        at liquibase.changelog.ChangeSet.execute(ChangeSet.java:670)
        at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:49)
        at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:111)
        at liquibase.Scope.lambda$child$0(Scope.java:160)
        at liquibase.Scope.child(Scope.java:169)
        at liquibase.Scope.child(Scope.java:159)
        at liquibase.Scope.child(Scope.java:138)
        at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:110)
        at liquibase.Scope.lambda$child$0(Scope.java:160)
        at liquibase.Scope.child(Scope.java:169)
        at liquibase.Scope.child(Scope.java:159)
        at liquibase.Scope.child(Scope.java:138)
        at liquibase.Scope.child(Scope.java:222)
        at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:94)
        at liquibase.Scope.lambda$child$0(Scope.java:160)
        at liquibase.Scope.child(Scope.java:169)
        at liquibase.Scope.child(Scope.java:159)
        at liquibase.Scope.child(Scope.java:138)
        at liquibase.Scope.child(Scope.java:222)
        at liquibase.Scope.child(Scope.java:226)
        at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:66)
        ... 28 more
Caused by: java.lang.NullPointerException
        at liquibase.integration.commandline.Main$SecureLogFilter.isLoggable(Main.java:2093)
        at java.logging/java.util.logging.Handler.isLoggable(Handler.java:346)
        at java.logging/java.util.logging.StreamHandler.isLoggable(StreamHandler.java:237)
        at java.logging/java.util.logging.StreamHandler.publish(StreamHandler.java:194)
        at java.logging/java.util.logging.ConsoleHandler.publish(ConsoleHandler.java:95)
        at java.logging/java.util.logging.Logger.log(Logger.java:979)
        at java.logging/java.util.logging.Logger.doLog(Logger.java:1006)
        at java.logging/java.util.logging.Logger.logp(Logger.java:1283)
        at liquibase.logging.core.JavaLogger.log(JavaLogger.java:23)
        at liquibase.logging.core.CompositeLogger.log(CompositeLogger.java:29)
        at liquibase.logging.core.CompositeLogger.log(CompositeLogger.java:29)
        at liquibase.logging.core.AbstractLogger.info(AbstractLogger.java:50)
        at liquibase.logging.core.AbstractLogger.info(AbstractLogger.java:45)
        at liquibase.changelog.ChangeSet.execute(ChangeSet.java:636)
        ... 48 more`



┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-885) by [Unito](https://www.unito.io/learn-more)

--- a/README.md
+++ b/README.md
@@ -108,7 +108,7 @@ mongo-java-driver:3.12.7
 
 ### Adjust connection string
  
-Connection url can be adjusted here: [`db.connection.uri`](./src/test/resources/application-test.properties)
+Connection url can be adjusted here: [`url`](./src/test/resources/liquibase.properties)
 Run Integration tests by enabling `run-its` profile 
 
 ### Run integration tests

--- a/pom.xml
+++ b/pom.xml
@@ -95,12 +95,6 @@
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-core</artifactId>
-            <version>${jackson-core.version}</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
             <version>${jackson-core.version}</version>
             <scope>compile</scope>

--- a/src/main/java/liquibase/ext/mongodb/change/AbstractMongoChange.java
+++ b/src/main/java/liquibase/ext/mongodb/change/AbstractMongoChange.java
@@ -30,4 +30,14 @@ public abstract class AbstractMongoChange extends AbstractChange {
     public boolean supports(Database database) {
         return database instanceof MongoLiquibaseDatabase;
     }
+
+    @Override
+    public boolean generateStatementsVolatile(final Database database) {
+        return false;
+    }
+
+    @Override
+    public boolean generateRollbackStatementsVolatile(final Database database) {
+        return false;
+    }
 }

--- a/src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java
+++ b/src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java
@@ -88,6 +88,15 @@ public class MongoLiquibaseDatabase extends AbstractNoSqlDatabase {
         return MONGODB_PRODUCT_NAME;
     }
 
+    @Override
+    public String getSystemSchema() {
+        return "admin";
+    }
+
+    /*********************************
+     * Custom Parameters
+     *********************************/
+
     public Boolean getAdjustTrackingTablesOnStartup() {
 
         if (adjustTrackingTablesOnStartup != null) {
@@ -106,4 +115,6 @@ public class MongoLiquibaseDatabase extends AbstractNoSqlDatabase {
         return LiquibaseConfiguration.getInstance().getConfiguration(MongoConfiguration.class)
                 .getSupportsValidator();
     }
+
+
 }

--- a/src/main/java/liquibase/nosql/database/AbstractNoSqlDatabase.java
+++ b/src/main/java/liquibase/nosql/database/AbstractNoSqlDatabase.java
@@ -21,22 +21,16 @@ package liquibase.nosql.database;
  */
 
 import liquibase.CatalogAndSchema;
-import liquibase.change.Change;
 import liquibase.database.AbstractJdbcDatabase;
 import liquibase.database.Database;
 import liquibase.database.DatabaseConnection;
-import liquibase.database.ObjectQuotingStrategy;
 import liquibase.exception.DatabaseException;
 import liquibase.exception.LiquibaseException;
 import liquibase.exception.ValidationErrors;
-import liquibase.sql.visitor.SqlVisitor;
 import liquibase.statement.DatabaseFunction;
-import liquibase.statement.SqlStatement;
 import liquibase.structure.DatabaseObject;
 import lombok.NoArgsConstructor;
 
-import java.io.IOException;
-import java.io.Writer;
 import java.math.BigInteger;
 import java.util.Collections;
 import java.util.List;
@@ -134,11 +128,6 @@ public abstract class AbstractNoSqlDatabase extends AbstractJdbcDatabase impleme
     }
 
     @Override
-    public String escapeColumnName(final String catalogName, final String schemaName, final String tableName, final String columnName, final boolean quoteNamesThatMayBeFunctions) {
-        return null;
-    }
-
-    @Override
     public String escapeColumnNameList(final String columnNames) {
         return null;
     }
@@ -150,7 +139,7 @@ public abstract class AbstractNoSqlDatabase extends AbstractJdbcDatabase impleme
 
     @Override
     public boolean supportsCatalogs() {
-        return false;
+        return true;
     }
 
     @Override
@@ -212,12 +201,6 @@ public abstract class AbstractNoSqlDatabase extends AbstractJdbcDatabase impleme
     }
 
     @Override
-    public boolean isSafeToRunUpdate() throws DatabaseException {
-        //TODO: Add the check to not be admin, etc.
-        return false;
-    }
-
-    @Override
     public List<DatabaseFunction> getDateFunctions() {
         //TODO: proper implementation
         return Collections.emptyList();
@@ -249,11 +232,6 @@ public abstract class AbstractNoSqlDatabase extends AbstractJdbcDatabase impleme
     }
 
     @Override
-    public String correctObjectName(final String name, final Class<? extends DatabaseObject> objectType) {
-        return null;
-    }
-
-    @Override
     public boolean isFunction(String string) {
         return false;
     }
@@ -261,11 +239,6 @@ public abstract class AbstractNoSqlDatabase extends AbstractJdbcDatabase impleme
     @Override
     public int getDataTypeMaxParameters(String dataTypeName) {
         return 0;
-    }
-
-    @Override
-    public CatalogAndSchema getDefaultSchema() {
-        return null;
     }
 
     @Override
@@ -279,49 +252,9 @@ public abstract class AbstractNoSqlDatabase extends AbstractJdbcDatabase impleme
     }
 
     @Override
-    public ObjectQuotingStrategy getObjectQuotingStrategy() {
-        return null;
-    }
-
-    @Override
-    public void setObjectQuotingStrategy(final ObjectQuotingStrategy quotingStrategy) {
-        //TODO: implementation
-    }
-
-    @Override
     public boolean createsIndexesForForeignKeys() {
         //Not applicable
         return false;
-    }
-
-    @Override
-    public boolean getOutputDefaultSchema() {
-        return false;
-    }
-
-    @Override
-    public void setOutputDefaultSchema(final boolean outputDefaultSchema) {
-        //TODO: implementation
-    }
-
-    @Override
-    public boolean isDefaultSchema(final String catalog, final String schema) {
-        return false;
-    }
-
-    @Override
-    public boolean isDefaultCatalog(final String catalog) {
-        return false;
-    }
-
-    @Override
-    public boolean getOutputDefaultCatalog() {
-        return false;
-    }
-
-    @Override
-    public void setOutputDefaultCatalog(final boolean outputDefaultCatalog) {
-        //TODO: implementation
     }
 
     @Override

--- a/src/main/java/liquibase/nosql/lockservice/AbstractNoSqlLockService.java
+++ b/src/main/java/liquibase/nosql/lockservice/AbstractNoSqlLockService.java
@@ -183,7 +183,7 @@ public abstract class AbstractNoSqlLockService implements LockService {
         } finally {
             try {
                 database.rollback();
-            } catch (DatabaseException e) {
+            } catch (final DatabaseException e) {
                 getLogger().severe("Error on acquire change log lock Rollback.", e);
             }
         }

--- a/src/main/java/liquibase/nosql/parser/json/JsonNoSqlChangeLogParser.java
+++ b/src/main/java/liquibase/nosql/parser/json/JsonNoSqlChangeLogParser.java
@@ -15,7 +15,6 @@ import liquibase.Scope;
 import liquibase.changelog.ChangeLogParameters;
 import liquibase.changelog.DatabaseChangeLog;
 import liquibase.exception.ChangeLogParseException;
-import liquibase.exception.LiquibaseException;
 import liquibase.logging.Logger;
 import liquibase.parser.ChangeLogParser;
 import liquibase.parser.core.ParsedNode;
@@ -147,7 +146,7 @@ public class JsonNoSqlChangeLogParser implements ChangeLogParser {
         }
     }
 
-    private void loadChangeLogParametersFromFile(ChangeLogParameters changeLogParameters, ResourceAccessor resourceAccessor, DatabaseChangeLog changeLog, JsonNode property, ContextExpression context, Labels labels, Boolean global) throws IOException, LiquibaseException {
+    private void loadChangeLogParametersFromFile(ChangeLogParameters changeLogParameters, ResourceAccessor resourceAccessor, DatabaseChangeLog changeLog, JsonNode property, ContextExpression context, Labels labels, Boolean global) throws IOException {
         Properties props = new Properties();
         try (
                 InputStream propertiesStream = resourceAccessor.openStream(null, property.get("file").asText())) {

--- a/src/test/java/liquibase/ext/mongodb/TestUtils.java
+++ b/src/test/java/liquibase/ext/mongodb/TestUtils.java
@@ -48,8 +48,8 @@ public final class TestUtils {
     public static final String EMPTY_STRING = "";
     public static final String EMPTY_OPTION = EMPTY_STRING;
     public static final String OPTION_1 = "{\"opt1\":\"option 1\"}";
-    public static final String DB_CONNECTION_PATH = "db.connection.uri";
-    public static final String PROPERTY_FILE = "application-test.properties";
+    public static final String DB_CONNECTION_PATH = "url";
+    public static final String PROPERTY_FILE = "liquibase.properties";
     private static final String CMD_DROP_ALL_ROLES = "{dropAllRolesFromDatabase: 1, writeConcern: { w: \"majority\" }}";
     private static final String CMD_DROP_ALL_USERS = "{dropAllUsersFromDatabase: 1, writeConcern: { w: \"majority\" }}";
     //private static final String CMD_GET_ALL_ROLES = "{getAllRolesFromDatabase: 1,  showPrivileges:false, showBuiltinRoles: false }}";

--- a/src/test/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabaseTest.java
+++ b/src/test/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabaseTest.java
@@ -1,12 +1,15 @@
 package liquibase.ext.mongodb.database;
 
+import liquibase.CatalogAndSchema;
 import liquibase.configuration.LiquibaseConfiguration;
 import liquibase.database.DatabaseFactory;
+import liquibase.database.ObjectQuotingStrategy;
 import liquibase.ext.mongodb.configuration.MongoConfiguration;
 import lombok.SneakyThrows;
 import org.junit.jupiter.api.*;
 
 import static java.lang.Boolean.FALSE;
+import static liquibase.servicelocator.PrioritizedService.PRIORITY_DATABASE;
 import static org.assertj.core.api.Assertions.assertThat;
 
 @TestInstance(TestInstance.Lifecycle.PER_METHOD)
@@ -19,8 +22,9 @@ class MongoLiquibaseDatabaseTest {
     @BeforeEach
     void setUp() {
         resetServices();
-        database = (MongoLiquibaseDatabase) DatabaseFactory.getInstance()
-                .findCorrectDatabaseImplementation(new MongoConnection());
+
+        database = new MongoLiquibaseDatabase();
+
         configuration = LiquibaseConfiguration.getInstance()
                 .getConfiguration(MongoConfiguration.class);
     }
@@ -35,11 +39,14 @@ class MongoLiquibaseDatabaseTest {
         LiquibaseConfiguration.getInstance().reset();
     }
 
+    @SneakyThrows
     @Test
     void getDefaultDriver() {
+        database = (MongoLiquibaseDatabase) DatabaseFactory.getInstance()
+                .findCorrectDatabaseImplementation(new MongoConnection());
+
         assertThat(database.getDefaultDriver("mongodb://qwe")).isEqualTo(MongoClientDriver.class.getName());
         assertThat(database.getDefaultDriver("cosmos://qwe")).isNull();
-
     }
 
     @Test
@@ -97,4 +104,58 @@ class MongoLiquibaseDatabaseTest {
         assertThat(configuration.getSupportsValidator()).isTrue();
         assertThat(database.getSupportsValidator()).isFalse();
     }
+
+    @Test
+    void getPriority() {
+        assertThat(database.getPriority())
+                .isEqualTo(PRIORITY_DATABASE)
+        .isEqualTo(5);
+    }
+
+    @Test
+    void supportsCatalogs() {
+        assertThat(database.supportsCatalogs()).isTrue();
+    }
+
+    @Test
+    void supportsSchemas() {
+        assertThat(database.supportsSchemas()).isFalse();
+    }
+
+    @Test
+    void getSchemaAndCatalogCase() {
+        assertThat(database.getSchemaAndCatalogCase()).isEqualTo(CatalogAndSchema.CatalogAndSchemaCase.ORIGINAL_CASE);
+    }
+
+    @Test
+    void getObjectQuotingStrategy() {
+        assertThat(database.getObjectQuotingStrategy()).isEqualTo(ObjectQuotingStrategy.LEGACY);
+    }
+
+    @Test
+    void getDefaultCatalogName() {
+        database.setDefaultCatalogName("catalog1");
+        assertThat(database.getDefaultCatalogName()).isEqualTo("catalog1");
+        // when schema present still catalog returned
+        database.setDefaultSchemaName("schema1");
+        assertThat(database.getDefaultCatalogName()).isEqualTo("catalog1");
+    }
+
+    @Test
+    void getDefaultSchemaName() {
+        database.setDefaultSchemaName("schema1");
+        assertThat(database.getDefaultSchemaName()).isEqualTo("schema1");
+        // when catalog is defined catalog is returned
+        database.setDefaultCatalogName("catalog1");
+        assertThat(database.getDefaultSchemaName()).isEqualTo("catalog1");
+    }
+
+    @Test
+    void getDefaultSchema() {
+        database.setDefaultSchemaName("schema1");
+        database.setDefaultCatalogName("catalog1");
+        assertThat(database.getDefaultSchema()).extracting(CatalogAndSchema::getCatalogName, CatalogAndSchema::getSchemaName)
+                .containsExactly("catalog1", "catalog1");
+    }
+
 }

--- a/src/test/resources/liquibase.properties
+++ b/src/test/resources/liquibase.properties
@@ -1,0 +1,78 @@
+####     _     _             _ _
+##      | |   (_)           (_) |
+##      | |    _  __ _ _   _ _| |__   __ _ ___  ___
+##      | |   | |/ _` | | | | | '_ \ / _` / __|/ _ \
+##      | |___| | (_| | |_| | | |_) | (_| \__ \  __/
+##      \_____/_|\__, |\__,_|_|_.__/ \__,_|___/\___|
+##                  | |
+##                  |_|
+##
+##      The liquibase.properties file stores properties which do not change often,
+##      such as database connection information. Properties stored here save time
+##      and reduce risk of mistyped command line arguments.
+##      Learn more: https://www.liquibase.org/documentation/config_properties.html
+####
+####
+##   Note about relative and absolute paths:
+##      The liquibase.properties file requires paths for some properties.
+##      The classpath is the path/to/resources (ex. src/main/resources).
+##      The changeLogFile path is relative to the classpath.
+##      The url H2 example below is relative to 'pwd' resource.
+####
+# Enter the path for your changelog file.
+changeLogFile=generic-1-insert-people.json
+
+#### Enter the Target database 'url' information  ####
+url=mongodb://localhost:27017/test_db?socketTimeoutMS=1000&connectTimeoutMS=1000&serverSelectionTimeoutMS=1000
+
+# Enter the username for your Target database.
+#username: dbuser
+
+# Enter the password for your Target database.
+#password: letmein
+
+#### Enter the Source Database 'referenceUrl' information ####
+## The source database is the baseline or reference against which your target database is compared for diff/diffchangelog commands.
+
+# Enter URL for the source database
+#referenceUrl: jdbc:h2:tcp://localhost:9090/mem:integration
+
+# Enter the username for your source database
+#referenceUsername: dbuser
+
+# Enter the password for your source database
+#referencePassword: letmein
+
+#### Liquibase Pro Key Information ####
+# Enter your Liquibase Pro key here.
+# If you don't have one, visit https://download.liquibase.org/liquibase-pro-trial-request-form/ to start a free trial!
+# liquibaseProLicenseKey:
+
+# Logging Configuration
+# logLevel controls the amount of logging information generated. If not set, the default logLevel is INFO.
+# Valid values, from least amount of logging to most, are:
+#   OFF, ERROR, WARN, INFO, DEBUG, TRACE, ALL
+# If you are having problems, setting the logLevel to DEBUG and re-running the command can be helpful.
+logLevel: DEBUG
+
+# The logFile property controls where logging messages are sent. If this is not set, then logging messages are
+# displayed on the console. If this is set, then messages will be sent to a file with the given name.
+# logFile: liquibase.log
+
+
+
+#### Liquibase Hub Information ####
+### Liquibase Hub is a free secure SaaS portal providing status reporting, monitoring & insights
+### into your Liquibase database release automation.
+### https://hub.liquibase.com
+
+## Add your free Hub API key here
+# liquibase.hub.apikey:
+# liquibase.hub.mode:all
+
+
+
+
+##  Get documentation at docs.liquibase.com       ##
+##  Get certified courses at learn.liquibase.com  ##
+##  Get support at liquibase.com/support         ##

